example/l2fwd_simple/Makefile.am

```diff
@@ -4,9 +4,11 @@ bin_PROGRAMS = odp_l2fwd_simple
 
 odp_l2fwd_simple_SOURCES = odp_l2fwd_simple.c
 
+if PLATFORM_IS_LINUX_GENERIC
```

Comment:
Could we just replace `HAVE_PCAP` with e.g. `_ODP_PKTIO_PCAP` (we already have this as a define)? This way each platform which supports pcap based pktio (e.g. odp-dpdk) could use this file without modifications.

Reply:
Sure, will fix in V5